.github/workflows/ci-mlir.yml

```diff
@@ -104,6 +104,13 @@ jobs:
         export PATH=$PATH:${GITHUB_WORKSPACE}/llvm-project/build/bin/
         lit -v tests/filecheck/ -DCOVERAGE
 
+    - name: Run benchmarks
+      run: |
+        cd xdsl
+        # Add mlir-opt to the path
+        export PATH=$PATH:${GITHUB_WORKSPACE}/llvm-project/build/bin/
+        ./benchmarks/dialects/experimental/stencil/run-laplace-benchmark.sh 1
```

Comment:
Why don't you just change the `clang` alias before running this script?

Reply:
Do you mean the following?
```
alias clang=${GITHUB_WORKSPACE}/llvm-project/build/bin/clang 
./benchmarks/dialects/experimental/stencil/run-laplace-benchmark.sh
```

Reply:
Yes, that would have been my attempt.

Reply:
I tried this earlier, but the alias is not followed from inside the script. Do you know of any way to circumvent this?

Reply:
That goes beyond my bash knowledge. Google might know something? O/w the curretn things is okay, I guess.